ctrlf.el

```diff
@@ -424,6 +426,14 @@ direction is backwards."
         (previous-history-element 1))
     (ctrlf-previous-match)))
 
+(defun ctrlf--symbol-at-point ()
+  "Return symbol at point.
+When doing regexp search, wrap it with \"\\_<\" and \"\\_>\"."
+  (let ((symbol (thing-at-point 'symbol t)))
+    (if ctrlf--regexp-p
+        (concat "\\_<" (regexp-quote symbol) "\\_>")
+      symbol)))
```

Comment:
This seems a little inconsistent to me between the regexp and non-regexp modes, but I guess it's the best we can do :/

Reply:
I agree. I think most of the case we want to find exactly the symbol at point, so the behavior in regexp search is right, while we have to compromise in literal search.